draft-ietf-rtcweb-jsep.xml

```diff
@@ -2384,14 +2384,21 @@ candidate:1 1 UDP 1694498815 192.0.2.33 10000 typ host
               mismatches between "c=" and the candidate lines;
               because JSEP always uses ICE, "ice-mismatch" is not useful in this context.
               </t>
-              <t>TODO</t>
+              <t>The "i=", "u=", "e=", "p=", "t=", "r=", "z=", "k=",
+	      lines are not used by this specification and MAY be ignored. </t>
             </list>
           </t>
           <t>The remaining lines are processed as follows:
             <list>
               <t>The "b=" line, if present, MUST be parsed as specified in
                 <xref target="RFC4566"/>, Section 5.8, and the bwtype and
-                bandwidth values stored.</t>
+              bandwidth values stored.</t>
+	      <t> The "v="  MUST have a version of 0.  </t>
```

Comment:
Did you mean to say "line"?

Also, these should probably be sorted in the order in which they appear in SDP.